ui-react/src/components/topBar/LogsCanvas.tsx

```diff
@@ -18,8 +20,8 @@ export default function LogsCanvas({ show, handleClose }: LogsCanvasProps) {
       <Offcanvas.Header closeButton>
         <Offcanvas.Title>לוגים:</Offcanvas.Title>
       </Offcanvas.Header>
-      <Offcanvas.Body>
-        [2022-12-07 12:34:27.432] [info] Welcome to caspion log
+      <Offcanvas.Body dir="ltr">
+        {lastLines}
```

Comment:
You should `.replaceAll('\r\n', '<br />')`

Reply:
both _replaceAl_l and _'<br/>'_ did not work for me well.
so I Implement your good idea for line separating in a quite different way.